file/1.3/modules/ROOT/pages/file-troubleshooting.adoc

```diff
@@ -33,6 +33,39 @@ To resolve this error:
 
 For additional information about enabling verbose logging for SDK components in your Mule application, see the xref:mule-sdk::troubleshooting.adoc[troubleshoot SDK components] documentation.
 
+== Understand Common Throws
+
+Here is a list of common throw messages and how to interpret them:
+
+* FILE:CONNECTIVITY
+
+The connection to the file server is no longer valid. Apply a reconnection policy if defined in the application setup.
+
+* FILE:ILLEGAL_PATH
+
+This error occurs if the *createParentDirectories* property is set to false and the target path doesn’t exist nor does its parent.
+
+* FILE:FILE_ALREADY_EXISTS
+
+This error occurs when the target file already exists and the *overwrite* property is set to false.
+
+* FILE:RETRY_EXHAUSTED
+
+The maximum number of retries for the operation is reached.
+
+* FILE:ACCESS_DENIED
+
+A File Connector operation attempted to access a file or a path and performed an action for which the user does not have permission.
+
+* FILE:FILE_LOCK
+
+An attempt to lock a file was unsuccessful.
+
+* FILE:ILLEGAL_CONTENT
+
+The *Write* operation received invalid content to write.
```

Comment:
Does this mean the same thing?
```suggestion
The *Write* operation was attempted using invalid content.
```
I think this might be one of those cases in which passive voice makes the meaning more clear.